### PR TITLE
Quick fix for simplecrf imports

### PR DESCRIPTION
**Pull request by masadcv**
Signed-off-by: masadcv <muhammad.asad@kcl.ac.uk>

Work around for #605 until the error is debugged.

### CODE REVIEW

**Comment by masadcv** [comment]
@SachidanandAlle I made all SImpleCRF imports optional. Please give it a try by uninstalling SimpleCRF and running any app. On my end this exp runs fine.

File: monailabel/scribbles/transforms.py [hunk 1]
```diff
@@ -31,6 +30,10 @@
     maxflow3d,
 )
 
+densecrf, _ = optional_import("dennseCRF")
+densecrf3d, _ = optional_import("denseCRF3D")
```
Comment (rijobro) [inline monailabel/scribbles/transforms.py:34]: here, you could use:
```suggestion
densecrf3d, has_densecrf3d = optional_import("denseCRF3D")
```
and then you could use `has_densecrf3d` elsewhere to display an error if the user tries to use scribbles.

Comment (masadcv) [reply monailabel/scribbles/transforms.py:34]: thanks! will update this.

Comment (rijobro) [reply monailabel/scribbles/transforms.py]: Looks good!

**Review by SachidanandAlle — APPROVED** [review]

**Comment by SachidanandAlle** [comment]
This solves one part of the problem... when SimpleCRF is installed by import fails due to xyz reasons...
I guess there is another thread where installation itself can be simplified with pre-built binaries while installing SimpleCRF package

**Pull request merged**